dataprofiler/tests/profilers/profiler_options/test_profiler_options.py

```diff
@@ -30,6 +30,8 @@ def test_default_profiler_options(self, *mocks):
                 self.assertFalse(profile.options.properties[column].is_enabled)
             elif column == "null_values" or column == "column_null_values":
                 self.assertIsNone(profile.options.properties[column])
+            elif column == "sampling_ratio":
```

Comment:
I think we need to also add tests below in the validate to ensure errors are done properly. Do we also need test to set?

Reply:
never mind, this isn't the spot!